docs/source/customizations/overview.mdx

```diff
@@ -276,6 +276,23 @@ flowchart LR;
 
 Additionally, some requests and responses may happen multiple times for the same operation. With subscriptions, for example, a subgraph sends a new `SubgraphResponse` whenever data is updated. Each response object travels through all the services in the response path and interacts with any customizations you've created.
 
+### Request and Response Buffering
```

Comment:
```suggestion
### Request and response buffering
```

Reply:
I capitalised both because `Request` and `Response` are capitalisable entities. We could take the uppercase 'B' from buffering though...

Reply:
done: 5b170d9e7828139d02615bd9400029141d307b80